gnomad/sample_qc/pipeline.py

```diff
@@ -179,7 +181,12 @@ def get_qc_mt(
     )
 
     if ld_r2 is not None:
-        qc_mt = qc_mt.persist()
+        if checkpoint_path:
+            logger.info("Checkpointing the MT and ld pruning")
+            qc_mt = qc_mt.checkpoint(checkpoint_path, overwrite=True)
+        else:
+            qc_mt = qc_mt.persist()
+            logger.info("Persisting the MT and ld pruning")
```

Comment:
Might also want to move before the actual persist
```suggestion
            logger.info("Persisting the MT and LD pruning")
```